lib/src/widget/tooltip/widget/tooltip.dart

```diff
@@ -14,6 +14,11 @@ typedef MessageFactory = Map<TooltipKey, TooltipParams>;
 typedef OnController = void Function(TooltipController);
 typedef ShowTooltipWhen = bool Function();
 
+enum TooltipStyle {
+  normal,
+  arrow,
```

Comment:
❓ will this `arrow` show always just down? if so, then please rename it.

Reply:
Yes, it's arrow down for now.